src/views/SignBtcTransactionLedger.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="container">
+    <div v-if="request.kind === 'sign-btc-transaction'" class="container">
```

Comment:
Is this necessary?

Reply:
Yes, request.kind can be something different, if the original request was different.
See `SignBtcTransaction.vue`.

Reply:
Oh i see, alright :)